source/components/organisms/SummaryList/SummaryList.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import GroupedList from '../../molecules/GroupedList/GroupedList';
+import SummaryListItem from './SummaryListItem';
+
+export interface Item {
+  title: string;
+  id: string;
+  type: 'number' | 'text' | 'date' | 'arrayNumber' | 'arrayText' | 'arrayDate';
+  category?: string;
+  inputId?: string;
+}
+
+interface Category {
```

Comment:
Would like to se prefix on this ie ISummaryListCategoryType. If Category is not a common type

Reply:
Renamed to SummaryListCategory